Haskell/syntax_test_haskell.hs

```diff
@@ -145,6 +163,62 @@
    countTheBeforeVowel = undefined
 
 
+--KEYWORDS
+
+import qualified Data.Vector.Mutable as MutableVector
+-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.import.haskell
+-- ^^^ keyword.control.import.haskell
+--     ^^^^^^^^^ keyword.control.import.haskell
+--               ^^^^^^^^^^^^^^^^^^^ support.other.module.haskell
+--                                   ^^ keyword.control.import.haskell
+--                                      ^^^^^^^^^^^^^ support.other.module.haskell
```

Comment:
This scoping is inconsistent with Scala, which has a similar import mechanism and which is itself consistent with JavaScript, Python, and C#. For reference:

```scala
   import Data.Vector.{Mutable => MutableVector}
// ^^^^^^ meta.import.scala keyword.other.import.scala
//        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.import.scala
//                    ^ meta.import.scala meta.import.selector.scala punctuation.section.group.begin.scala
//                     ^^^^^^^^^^^^^^^^^^^^^^^^^ meta.import.scala meta.import.selector.scala
//                             ^^ meta.import.scala meta.import.selector.scala keyword.operator.arrow.scala
```